HopsanGUI/Dialogs/OptimizationDialog.cpp

```diff
@@ -1098,6 +1113,25 @@ void OptimizationDialog::generateDifferentialEvolutionScript()
     mScript = templateCode;
 }
 
+void OptimizationDialog::generateGeneticScript()
+{
+    QFile templateFile(gpDesktopHandler->getExecPath()+"../Scripts/HCOM/optTemplateGenetic.hcom");
```

Comment:
Dont we have a getScripts path (or similar) not sure. Also no checking if file exists. (It should, but ......)

Reply:
No, the getScripts function returns the scripts folder under Documents. We want the one in the installation directory here.